onnxruntime/test/optimizer/graph_transform_test_builder.cc

```diff
@@ -20,6 +21,90 @@
 namespace onnxruntime {
 namespace test {
 
+static std::vector<std::byte> GetZeroPointBytes(int64_t zero_point, ONNX_NAMESPACE::TensorProto_DataType type) {
+  switch (type) {
+    case ONNX_NAMESPACE::TensorProto_DataType_INT8: {
+      int8_t val = static_cast<int8_t>(zero_point);
+      auto span = ReinterpretAsSpan<const std::byte, const int8_t>(gsl::make_span(&val, 1));
```

Comment:
would using gsl::span::as_bytes be simpler?

Reply:
Yes, good call. I updated this.